### PR TITLE
Feature simplified grid

### PR DESCRIPTION
**Pull request by Bhakti199**
here I added simplified grid - grid-2 , grid-3 and grid-auto-fit

### CODE REVIEW

File: simplified-grid/simplified-grid.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@import url(/style.css);
+
+.grid-container {
+  border: 1px solid var(--dark-black);
+  width: 60%;
+  height: auto;
+  margin: 2rem 0;
+}
+.grid-item {
+  padding: 10px;
```
Comment (shaksham08) [inline simplified-grid/simplified-grid.css:10]: use rem instead of px if not necessary

**Review by shaksham08 — CHANGES_REQUESTED** [review]
Looks good. Added one comment

File: simplified-grid/simplified-grid.css (hunk 1 above)
Comment (Bhakti199) [reply simplified-grid/simplified-grid.css:10]: Its a small value, I am assuming it to not to change in responsiveness too. Thats why used px.

Comment (shaksham08) [reply simplified-grid/simplified-grid.css:10]: oh cool then

**Review by shaksham08 — APPROVED** [review]

**Review by ozairkhan2810 — APPROVED** [review]
looks good, just maintain a blank line between blocks for readability
good to merge

**Pull request merged**